src/main/java/seedu/address/logic/commands/CommandTrie.java

```diff
@@ -0,0 +1,116 @@
+package seedu.address.logic.commands;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Class for the Trie of possible commands.
```

Comment:
I think might be better not to use the word "Class" for the header comment, as it does not really indicates what it is use for.

Don't have to line break so soon, can let the comment reach until 120char before breaking.